### DISCUSSION TITLE
On Tailwind CSS arbitrary values | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#64] [Announcements]
# On Tailwind CSS arbitrary values | Leo's dev blog – stories, insights, and ideas

How to use arbitrary values in Tailwind CSS to create custom styles for your components. Some nice tips to make your Tailwind CSS workflow more efficient.

https://www.leohuynh.dev/blog/on-tailwind-css-arbitrary-values

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
